src/shared_modules/rsync/testtool/config.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
I think we should move this config.json and input.json files into a new "input" folder to separate implementation and input data files